app/helpers/multi_languages_helper.rb

```diff
@@ -0,0 +1,146 @@
+module MultiLanguagesHelper
+
+  def portal_language_help_text
```

Comment:
`portal_language_help_text`, `portal_languages` and `portal_language_selector` can be removed as not used